types/src/proposals.ts

```diff
@@ -254,8 +250,6 @@ export class ProposalDetails extends JoyEnum({
   RuntimeUpgrade: Bytes,
   SetElectionParameters: ElectionParameters,
   Spending: SpendingParams,
-  SetLead: SetLead,
-  SetContentWorkingGroupMintCapacity: u128,
```

Comment:
So I guess we will keep these and rename to with `Deprecated` prefix. As per 

https://github.com/Joystream/joystream/pull/1510/files#diff-d826bd63d452cdb624a717e8355d24a3R58

And same for other deprecated variants: `EvictStorageProvider`, `SetStorageRoleParameters`